app/middleware/postcodeLookup.js

```diff
@@ -0,0 +1,60 @@
+const postcodeValidator = require('../lib/postcodeValidator');
+const log = require('../lib/logger');
+const PostcodesIOClient = require('postcodesio-client');
+
+// rewire (a framework for mocking) doesn't support const
+// eslint-disable-next-line no-var
+var PostcodesIO = new PostcodesIOClient();
+// eslint-disable-next-line no-var
+var renderer = require('./renderer');
+
+function outsideEngland(outcodeDetails) {
+  return !outcodeDetails.country.some(c => c === 'England');
+}
+
+function lookupPostcode(req, res, next) {
+  const postcode = res.locals.postcode;
+
+  if (postcodeValidator.isOutcode(postcode)) {
+    PostcodesIO.outcode(postcode, (err, outcodeDetails) => {
+      if (outcodeDetails && outsideEngland(outcodeDetails)) {
+        log.info('validate-outcode-notEnglish');
```

Comment:
Is this logging needed and if so does it need to be logged at `info`? Looks like it is duplicated too.